src/components/CreateCampaign.js

```diff
@@ -289,7 +289,7 @@ class CreateCampaign extends React.Component {
         formData.append(
             "myFile",
             this.state.mainImageFile,
-            imgID,
+            `${imgID}.${fileType}`,
```

Comment:
what happens when a user tries uploading a file w/o an extension?

Reply:
that is checked on line 278:
 if(!this.state.mainImageFile || !this.state.mainImageFile.type) {